client-linuxapp/src/serviceinfo.rs

```diff
@@ -69,6 +74,54 @@ fn create_user(user: &str) -> Result<()> {
     Ok(())
 }
 
+fn create_user_with_password(user: &str, password: &str) -> Result<()> {
+    // Checks if user already present
+    let user_info = passwd::Passwd::from_name(user);
+    if user_info.is_some() {
+        log::info!("User {} is already present", user);
+        return Ok(());
+    }
+    
+    log::info!("Encrypting password");
+    let default_params: Sha256Params = Default::default();
+    let str_encrypted_pw = sha256_simple(password, &default_params).expect("Hashing failed");
```

Comment:
don't forget the `?` after the `expect`, we want to propagate the errors to the caller function

Reply:
I tried to add in the `?` after `.expect("Hashing failed")` but I was given the error below. I can put more time into it once I get ssh working again. 
```
error[E0277]: the `?` operator can only be applied to values that implement `Try`
  --> client-linuxapp/src/serviceinfo.rs:87:28
   |
87 |     let str_encrypted_pw = sha256_simple(password, &default_params).expect("Hashing failed")?;
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `String`
   |
   = help: the trait `Try` is not implemented for `String`
   ```

Reply:
I see :thinking: , all good then